content/en/blog/gitops-your-falco-rules/index.md

```diff
@@ -0,0 +1,805 @@
+  repository: falcosecurity/rules/falco-rules
+  description: Falco rules that are loaded by default
+  home: https://github.com/falcosecurity/rules/blob/main/rules/falco_rules.yaml
+  keywords:
+    - falco-rules
+  license: apache-2.0
+  maintainers:
+    - email: cncf-falco-dev@lists.cncf.io
+      name: The Falco Authors
+  sources:
+    - https://github.com/falcosecurity/rules/blob/main/rules/falco_rules.yaml
+
+```
+
+What we can see in that index are the locations of the different OCI artifacts containing each one of the rules files. That in particular is the entry of the **falco-rules** artifact. However, we see no trace of that **:0** suffix.
+
+Let's use the command `falcoctl artifact info` to see where that takes us. That command will use the **index** file and will share information obtained from the remote artifact.
+```
+$ falcoctl artifact info falco-rules
+
+REF                                    	TAGS                            
+ghcr.io/falcosecurity/rules/falco-rules	latest, 0.1.0, 0.1, 0, 0.1.0-rc1
+```
+
+We found it! The **:0** suffix is just a floating tag referring, in our example, to the **0.1.0** tag. Falco could have used the **0.1.0** tag directly, the **0.1** one, or even the **:latest** tag (but that could have had some consequences in the future).
+
+Summarizing what we have learned so far:
+- `falcoctl` will follow the artifact tagged as **falco-rules:0** which resolves to the OCI artifact _ghcr.io/falcosecurity/rules/falco-rules:0.1.0_. 
+- This information results from combining the `/etc/falcoctl/falcoctl.yaml` configuration file and the `https://falcosecurity.github.io/falcoctl/index.yaml` index file. 
+- The OCI artifact _ghcr.io/falcosecurity/rules/falco-rules:0_ could point to a different artifact in the future without us having to reconfigure every Falco installation. 
+- Every 6 hours, `falcoctl` will check whether the artifact has changed and will update it locally.
+
+#### Customize Falcoctl configuration
+
+To let `falcoctl` manage our new artifacts, we are going to add some changes to our `falcoctl.yaml` configuration. 
+
+First, we are installing new custom rules, so we'll add that to the **artifact.install** section:
+
+```
+artifact:
+...
+  install:
+    rulesfilesdir: /etc/falco/rules.d/
+    refs:     
+      - ghcr.io/vjjmiras/cool-falco-rules/custom-rules:main
+```
+
+Then, let's add the new artifact to follow and decrease the checking interval to 5 minutes:
+
+```
+artifact:
+  follow:
+    every: 5m0s
+    ...
+    rulesfilesdir: /etc/falco/rules.d
+    refs:
+      # Leave out the predefined ruleset for the moment
+      # - falco-rules:0
+      - ghcr.io/vjjmiras/cool-falco-rules/custom-rules:main
+```
+
+{{% pageinfo color="info" %}}
+ 
+The reason why we are leaving out the `falco-rules:0` ruleset is that it exists in the `/etc/falco` directory and we want to download the followed artifacts to the `/etc/falco/rules.d` directory, so we don't have to apply any extra configuration to falco.
```

Comment:
This explanation is not clear to me

Reply:
Ok, I opted to modify the falco.yaml configuration to make this more clear.